### PR TITLE
GO-8084:3rd party library react-native-geocoder-reborn does not support non-English address

### PR DESCRIPTION
**Pull request by yuhui-zhao**
Different regions have different formats for the address. For example, the address will be in the country, state, city, street, and zip in Chinese. The ios API geocodeAddressString expects the address string in the correct format. 

Solution: Add a new API to take the address object as the parameter and let ios output the address string based on the device's region setting.

### CODE REVIEW

**Comment by jiwansvmx** [comment]
@yuhui-zhao Instead of all these changes to setup full CI in a fork, you should create a pr upstream and see if they merge it.

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+
+        stage('Install Dependencies') {
+            sh './jenkins.sh install'
+        }
+
+        stage('Lint') {
+            //timeout after 20 minutes in Test stage
+            timeout(time: 20, unit: 'MINUTES') {
+                print "Lint is NOT yet implemented!"
+                //sh './jenkins.sh flow'
```
Comment (ezengsvmx) [inline Jenkinsfile]: Why keep a big block of commented code?

Comment (yuhui-zhao) [reply Jenkinsfile]: the whole file copied from another project, leave it as a reference now. Only Robert is clear if it is needed or not in the future.

**Comment by yuhui-zhao** [comment]
@jiwansvmx I will do it later.

**Pull request merged**